app/state/app/index.ts

```diff
@@ -18,6 +19,7 @@ const initialState: AppState = {
   stakingRebase: undefined,
   treasuryBalance: undefined,
   rebaseBlock: undefined,
+  showCheckURLBanner: true,
```

Comment:
@biwano hmm If changing the language causes the banner to reappear, this means that the `Home` component is fully unmounting and remounting.

A full remount miiight cause problems with other states and `useEffects` too (fetching app data etc, etc,). We should be able to just switch the language without remounting everything...

Plus it would be nice to keep single-use states like this out of redux.

Can you investigate:
1. confirm _app.tsx component is _not_ remounting, and confirm that `Home` and other child components are remounting? This means it's the i18n provider causing the re-mount.
2. see if we can prevent it somehow?

Reply:
1. I confirmed that the i18n provider is causing the re-mount
2. Great advice! I could prevent it this way: 
```    <I18nProvider i18n={i18n} forceRenderOnLocaleChange={false}>
```